server/src/main/java/org/apache/druid/segment/indexing/granularity/UniformGranularitySpec.java

```diff
@@ -139,10 +138,18 @@ public GranularitySpec withIntervals(List<Interval> inputIntervals)
     return new UniformGranularitySpec(segmentGranularity, queryGranularity, rollup, inputIntervals);
   }
 
+  @Override
```

Comment:
You can move up this to the abstract base class

Reply:
Good idea. Done.